endpoints/openrtb2/auction.go

```diff
@@ -765,7 +769,7 @@ func (deps *endpointDeps) validateRequest(req *openrtb_ext.RequestWrapper, isAmp
 		}
 	}
 
-	var aliases map[string]string
+	var requestAliases map[string]string
```

Comment:
Making clear these are request aliases, not hardcoded aliases. Request alias names are still case sensitive.